server/src/main/java/io/unitycatalog/server/service/Scim2UserService.java

```diff
@@ -94,19 +99,35 @@ public HttpResponse createScimUser(UserResource userResource) {
       return HttpResponse.ofJson(asUserResource(user));
     } catch (BaseException e) {
       if (e.getErrorCode() == ErrorCode.NOT_FOUND) {
+        String pictureUrl = "";
+        if (userResource.getPhotos() != null && !userResource.getPhotos().isEmpty()) {
+          pictureUrl = userResource.getPhotos().get(0).getValue().toString();
+        }
         User user =
-            USER_REPOSITORY.createUser(
-                new CreateUser()
-                    .name(userResource.getDisplayName())
-                    .email(primaryEmail.getValue())
-                    .externalId(userResource.getExternalId()));
+            USER_REPOSITORY
+                .createUser(
+                    new CreateUser()
+                        .name(userResource.getDisplayName())
+                        .email(primaryEmail.getValue())
+                        .externalId(userResource.getExternalId()))
+                .pictureUrl(pictureUrl);
         return HttpResponse.ofJson(asUserResource(user));
       } else {
         throw e;
       }
     }
   }
 
+  @Get("/current")
```

Comment:
This is fine, although it's flexing the SCIM spec a little bit, but we can let it roll. I guess since the current user is just in the `Authorization` header, there's no way to get at it from the browser so the only choice is to ask the server who we are.